src/Translator/DummyTranslator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Validator\Translator;
+
+use Laminas\I18n\Translator\TranslatorInterface as I18nTranslatorInterface;
+
+final class DummyTranslator implements I18nTranslatorInterface
```

Comment:
```suggestion
/** @internal */
final class DummyTranslator implements I18nTranslatorInterface
```

I'm assuming that this can be marked internal - if it can, it should be.